### PR TITLE
feat(#99): add  couch2pg exporter compose file to run stand alone without watchdog

### PR DESCRIPTION
**Pull request by mrjones-plip**
This PR adds YAML files to allow watchdog to monitor a remote postgres instances for couch2pg backlog but not need any postgress credentials or have an SSH tunnel to the server.  see https://github.com/medic/cht-docs/pull/1378 for steps to test

per #99

### CODE REVIEW

**Review by jkuester — APPROVED** [review]
My only concern around this approach is with the extra complexity of remembering which compose files to include when running Docker commands.  :sweat: However, I think this is the right approach and we should look into simplifying docker commands in a separate task: https://github.com/medic/cht-watchdog/issues/103

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ grafana.ini
 delete-rules.yml
 build
 exporters/postgres/sql_servers.yml
+data-ingest
```
[the remark this answers is not in the document]
Comment (mrjones-plip) [reply .gitignore:14]: oop!  this should indeed be there, but maybe not in this PR - sorry!  It's from the [data ingest repo for watchdog ](https://github.com/medic/cht-app-monitoring-data-ingest/tree/main/watchdog-config).

File: exporters/postgres/scrape-only.yml
[the remark this answers is not in the document]
Comment (mrjones-plip) [reply exporters/postgres/scrape-only.yml:1]: good thinking! fixed in 9e2460d and then I'll go update [the docs](https://github.com/medic/cht-docs/pull/1378) too

File: exporters/postgres/scrape-custom.yml
[the remark this answers is not in the document]
Comment (mrjones-plip) [reply exporters/postgres/scrape-custom.yml:1]: d'oh!  Good catch.  It [is in there already](https://github.com/medic/cht-watchdog/blob/9e2460d773260867372421808b1260561edae18e/.gitignore#L15), but maybe after I committed the YAML file?

lessee there....

[the remark this answers is not in the document]
Comment (mrjones-plip) [reply exporters/postgres/scrape-custom.yml:1]: k - fixed in 8b86b9b

**Pull request merged**